app/institutions/dashboard/-components/institutional-users-list/styles.scss

```diff
@@ -96,21 +90,122 @@
     margin-left: 4px;
 }
 
-th .sort-arrow {
+.sort-arrow {
     display: inline-block;
     vertical-align: middle;
     color: #fff;
 }
 
-.table-filters {
+.select-container {
+    width: 100%;
+    display: flex;
+    justify-content: flex-end;
+    float: right;
+}
+
+.select {
+    min-width: 120px;
+    padding: 7px 16px 7px 14px;
+    border-color: #ddd;
+    border-radius: 2px;
+    color: #337ab7;
+    text-align: center;
+    margin: 15px;
+
+    span {
+        margin-left: 0;
+    }
+}
+
+.filter-container {
     display: flex;
     justify-content: flex-end;
     align-items: center;
     width: 100%;
-    padding: 10px 0;
+}
+
+.dropdown-trigger {
+    background-color: #fff;
+    border: 1px solid #ddd;
+    color: #007bff;
+    border-radius: 4px;
+    padding: 9px;
+    font-size: 14px;
+    cursor: pointer;
+    display: flex;
+    align-items: center;
+    justify-content: space-between;
+    box-shadow: 0 1px 2px rgba(0, 0, 0, 0.05);
+}
+
+.dropdown-trigger:hover {
+    background-color: #f8f9fa;
+    border-color: #007bff;
+}
+
+.dropdown-panel {
+    position: absolute;
+    top: calc(100% + 5px);
+    right: 0;
+    background-color: #fff;
+    border: 1px solid #ddd;
+    border-radius: 4px;
+    padding: 15px;
+    box-shadow: 0 4px 8px rgba(0, 0, 0, 0.1);
+    width: 220px;
+}
+
+.dropdown-content {
+    display: flex;
+    flex-direction: column;
     margin-bottom: 10px;
 }
 
+.dropdown-content label {
+    display: flex;
+    align-items: center;
+    padding: 4px 0;
+    font-size: 14px;
+    font-weight: lighter;
+}
+
+.dropdown-content [type='checkbox'] {
+    margin-right: 8px;
+    cursor: pointer;
+}
+
+.dropdown-actions {
+    display: flex;
+    justify-content: flex-end;
+    padding-top: 10px;
+    border-top: 1px solid #eee;
+}
+
+.dropdown-actions button {
+    min-width: 80px;
+    padding: 6px 12px;
+    border-radius: 4px;
+    font-size: 14px;
+    cursor: pointer;
+    margin-left: 10px;
+}
+
+.apply-button {
+    background-color: #007bff;
+    color: #fff;
+    border: 1px solid transparent;
+    padding: 6px;
+
+}
+
+.apply-button:hover {
+    background-color: #0056b3;
+}
```

Comment:
I think you'll be able to remove a good chunk of these styles if you change the "Cancel" and "Apply" buttons to use `<Button>`s. You may be able to remove the `.apply-button` class entirely and most of `.dropdown-actions button`. I suspect for `.dropdown-actions button` you still may need `min-width`, `margin-left` and possible `padding`